modules/openapi-generator/src/main/resources/python-flask/requirements.mustache

```diff
@@ -1,4 +1,7 @@
-connexion >= 2.0.2
+connexion >= 2.5.0; python_version>="3.6"
+connexion >= 2.3.0; python_version=="3.5"
+connexion >= 2.3.0; python_version=="3.4"
+connexion == 2.4.0; python_version<="2.7"
```

Comment:
This is needed because connexion was not correctly omitting older versions of python with python_requires
See https://github.com/zalando/connexion/pull/1109